demos/slider.html

```diff
@@ -65,6 +65,7 @@ <h2>Continuous Slider</h2>
               aria-valuemin="0" aria-valuemax="50" aria-valuenow="20"
               aria-label="Select Value">
                 <svg class="mdc-slider__thumb-handle" width="34" height="34">
+                  <circle class="mdc-slider__thumb-handle-disabled" cx="17" cy="17" r="6"></circle>
```

Comment:
Is there a reason for this being the same size and position as the `<circle>` below?

Reply:
Same as above comment.